why.js

```diff
@@ -205,6 +206,59 @@ module.exports = function whyNotEqual(value, other) {
 		if (isProto.call(other, value)) { return 'second argument is the [[Prototype]] of the first'; }
 		if (getPrototypeOf(value) !== getPrototypeOf(other)) { return 'arguments have a different [[Prototype]]'; }
 
+		var valueIsFn = typeof value === 'function';
+		var otherIsFn = typeof other === 'function';
+
+		if (!valueIsFn || !otherIsFn) {
```

Comment:
why this line? surely either a function or an object could both have toStrings or valueOfs that need comparing?

Reply:
otherwise `functions with same arity/name/body are equal despite whitespace between signature and body` will fail because:

```js
> const foo = Object(function(){});
undefined
> const bar = Object(function () {});
undefined
> foo.toString()
'function(){}'
> bar.toString()
'function () {}'
```

our options are to include this line, or delete that test. including this line is more backwards compatible.

Reply:
aha, hmm. that means this new functionality wouldn't apply to functions, which is weird. at the least, let's document that in the readme?

an alternative, tho, would be to skip the check for a function whenever the function's `toString` is `===` to `Function.prototype.toString`?